### PR TITLE
update with Windows Server information

### PR DESCRIPTION
**Pull request by DevOpsStyle**
Hello Team,

worked with a Cx in order to enroll Windows Server devices through MDE in Intune. Windows Servers (no Domain Controller) after the configuration (reported in Microsoft doc) can take some hours to show correctly these devices. When devices are showed up MDM can take up to 2-3 days to load correctly the security configuration to the configured Windows Servers.

Tested on Cx and also on my personal Tenant (In total all the windows server loaded correctly the MDM security configuration after 3 days from enroll.

Thank you for your support

### CODE REVIEW

**Comment by Brenduns** [comment]
Checking with product team on this 2 to 3 day delay.

File: memdocs/intune/protect/mde-security-integration.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ When you select a policy, you'll see information about the device check-in statu
 
 ### Device check-in frequency
 
-Devices managed by this capability check in with Microsoft Endpoint Manager every 90 minutes to update policy.
+During the first Enrollment through MDE in Microsoft Endpoint Manager Windows Server devices will take up to 2-3 days to correctly load security policy for the first time. After the Enroll Devices managed by this capability check in with Microsoft Endpoint Manager every 90 minutes to update policy.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/mde-security-integration.md:69]:
```suggestion
During the first enrollment through Microsoft Defender for Endpoint in Microsoft Endpoint Manager, Windows Server devices will take up to two or three days to correctly load security policy for the first time. Thereafter, the enrolled devices managed by this capability check in with Microsoft Endpoint Manager every 90 minutes to update policy.
```

File: memdocs/intune/protect/mde-security-integration.md [hunk 2]
```diff
@@ -97,9 +97,14 @@ The following security settings are pending deprecation. The Security Management
 - Expedite telemetry reporting frequency (under **Endpoint Detection and Response**)
 - AllowIntrusionPreventionSystem (under **Antivirus**)
 
+### Managing security configurations on Windows Server (no domain Controller)
+
+The first Enrollment with MDE can take some hours to correctly show the Devices in Microsoft Endpoint Manager. After the enrollment the system will take up to 2-3 days to load for the first time security policy.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/mde-security-integration.md:102]:
```suggestion
### Managing security configurations on Windows Server (no domain controller)

The first enrollment with Microsoft Defender for Endpoint might take some hours to correctly show the devices in Microsoft Endpoint Manager. After the enrollment, the system will take up to two or three days to load security policy for the first time.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DevOpsStyle, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by DevOpsStyle** [comment]
Thank you @JohanFreelancer9. If the pr is approved and we have no other opened point can you close the pr with the "#sign-off" comment?

**Comment by JohanFreelancer9** [comment]
#signoff

**Comment by DevOpsStyle** [comment]
> @DevOpsStyle - I did the copy review, but only the assigned author can sign off on this PR.

Oh clear, thank you

**Comment by Brenduns** [comment]
@DevOpsStyle    Thanks for this PR submission. We cannot accept this however as after consulting with the product team, this length of dely is far from expected and not correct.   Unfortunately, via docs feedback, we cannot invetigate this but the recomendation is that should your enviroment be experiencing this type of delay to open a new support issue to have the source invesigated.

**Pull request closed without merging**